src/app/app.component.html

```diff
@@ -0,0 +1,40 @@
+<nav>
```

Comment:
@Fleischers The way I named files is an Angular CLI naming standard. I don't know why they made it so but I believe that if we have lots of components in a single directory it is much easier to determine which files relate to one or another component if they have the same names. It's just more visually clear. And we're not trying to convince anyone that CSS is a style and HTML is a template by writing it one more time in a filename. It's just a given. What else can it be?

And i'm not really happy with that main.browser.ts naming also. But I can live with that if we decide to. So it's just a matter of decision here.